pithos/pithos.py

```diff
@@ -228,7 +228,7 @@ def init_core(self):
         self.art_worker = GObjectWorker()
 
         theme = Gtk.IconTheme.get_default()
-        aa = theme.load_icon('pithos-album-default', 128, 0)
+        aa = theme.load_icon('pithos-album-default', ALBUM_ART_SIZE, 0)
 
         self.default_album_art = aa.scale_simple(ALBUM_ART_SIZE, ALBUM_ART_SIZE, GdkPixbuf.InterpType.BILINEAR)
```

Comment:
Yes after playing with it the scaler can be removed. I've also been playing with a little padding and a subtle shadow.

What do you think of this? If you like it I can amend this commit with the changes.
![screenshot from 2016-03-06 14-58-54](https://cloud.githubusercontent.com/assets/6667703/13556989/8384edce-e3ac-11e5-9f10-a0ddefcc25ab.png)